test/cli/uninitialized-explanation/uninitialized.rb

```diff
@@ -0,0 +1,16 @@
+# typed: true
+
+extend T::Sig
+
+sig {params(x: Integer).void}
+def takes_int(x); end
+
+def uninitialized_loc_is_decl_loc
+  if T.unsafe(nil)
+    x = 1
+  else
+    T.unsafe(self).raise "Unanalyzable raise"
```

Comment:
What's going on here? Is this different from omitting the `else` branch?

Reply:
Not sure, I just copied this example from #1909 :)